### PR TITLE
Register Subdomain: sameerahmed

### PR DESCRIPTION
**Pull request by sxmeer-ahmed**
Added the domain: sameerahmed

### CODE REVIEW

File: domains/sameerahmed.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+ 
+    {
+      "owner": {
+        "username": "sxmeer-ahmed",
+        "email": "ahmedkhansameer50@gmail.com"
+      },
+      "record": {
+        "TXT": "786"
```
Comment (WilliamDavidHarrison) [inline domains/sameerahmed.json:8]: Please provide a valid record

**Review by wdhdev — CHANGES_REQUESTED** [review]

File: domains/sameerahmed.json (hunk 1 above)
Comment (VaibhavSys) [reply domains/sameerahmed.json:8]: > Please provide a valid record

It is a valid record.

**Review by VaibhavSys — APPROVED** [review]

File: domains/sameerahmed.json (hunk 1 above)
Comment (WilliamDavidHarrison) [reply domains/sameerahmed.json:8]: > > Please provide a valid record
> 
> It is a valid record.

It isn't as it is not used for anything and it's their only record.

Comment (VaibhavSys) [reply domains/sameerahmed.json:8]: We don't know what he could be using it for, its valid as in DNS and he could be doing something of his own with that record.

Comment (WilliamDavidHarrison) [reply domains/sameerahmed.json:8]: > We don't know what he could be using it for, its valid as in DNS and he could be doing something of his own with that record.

We have to know what he's using it for, it's literally in the docs.

**Comment by WilliamDavidHarrison** [comment]
No changes that meet our requirements

**Pull request closed without merging**